mod-finance/examples/batch.sample

```diff
@@ -0,0 +1,74 @@
+{
+  "transactionsToCreate": [
+    {
+      "id": "c5732efb-9536-4a49-a22e-1ec6ca8a7922",
+      "amount": 25,
+      "awaitingPayment": {
+        "encumbranceId": "12ac70a4-5c9d-112f-9acb-852adf67d7e1",
+        "releaseEncumbrance": true
+      },
+      "currency": "USD",
+      "description": "PO_Line: History of Incas",
+      "fiscalYearId": "684b5dc5-92f6-4db7-b996-b549d88f5e4e",
+      "fromFundId": "7fbd5d84-62d1-44c6-9c45-6cb173998bbd",
+      "invoiceCancelled": false,
+      "source": "Invoice",
+      "sourceInvoiceId": "5925e966-f242-4842-9d5e-b33a651bb390",
+      "sourceInvoiceLineId": "ee60ee78-f487-4940-adcf-fa0726989823",
+      "transactionType": "Pending payment"
+    }
+  ],
+  "transactionsToUpdate": [
+    {
+      "id": "b7553168-60af-4e6d-bcd3-078199a0b353",
+      "amount": 25.17,
+      "currency": "USD",
+      "description": "PO_Line: History of Incas",
+      "encumbrance": {
+        "amountExpended": 1567.92,
+        "amountAwaitingPayment": 12.50,
+        "initialAmountEncumbered": 1720.50,
+        "status": "Unreleased",
+        "orderType":  "One-Time",
+        "orderStatus":  "Open",
+        "subscription": false,
+        "reEncumber": false,
+        "sourcePurchaseOrderId": "e5ae4afd-3fa9-494e-a972-f541df9b877e",
+        "sourcePoLineId": "c5ced39d-672f-3cba-a13a-ceb06a5d4c6b"
+      },
+      "fiscalYearId": "4438bc3b-4b4a-4e5e-9886-5f69b48a7a32",
+      "fromFundId": "27f27438-3197-4281-a353-511829421d6e",
+      "paymentEncumbranceId": "80c28fb4-c34e-48d6-b8c2-de6660c3e0aa",
+      "source": "Invoice",
+      "sourceFiscalYearId": "684b5dc5-92f6-4db7-b996-b549d88f5e4e",
+      "sourceInvoiceId": "b360db6c-b825-4d74-ac63-5be506892e2b",
+      "sourceInvoiceLineId": "2be1e551-be27-4b03-b770-30e5e8211d19",
+      "toFundId": "69640328-788e-43fc-9c3c-af39e243f3b7",
+      "transactionType": "Encumbrance",
+      "metadata": {
+        "createdDate": "2019-10-31T09:21:44.386+0000",
+        "createdByUserId": "02408630-bce2-5a82-b577-9be500e9b324",
+        "updatedDate": "2019-10-31T09:21:44.386+0000",
+        "updatedByUserId": "02408630-bce2-5a82-b577-9be500e9b324"
+      }
+    }
+  ],
+  "idsOfTransactionsToDelete": [
```

Comment:
lets rename to transactionIdsToDelete to have the same pattern to start any word with "transaction"

Reply:
That sounds a bit odd to me, as if you wanted to delete the ids. Initially I had simply "delete", but that made the Java mapping more complex (otherwise you would have a "getDelete" method, for instance), so I changed the property names again. "idsOfTransactionsToDelete" is lengthy but more explicit, and in Java you automatically get methods like "getIdsOfTransactionsToDelete()" which is clearly a List of String...

Reply:
okay got it , agree